lib/trento/application/integration/discovery/payloads/sap_system_discovery_payload.ex

```diff
@@ -343,6 +341,10 @@ defmodule Trento.Integration.Discovery.SapSystemDiscoveryPayload do
     defp parse_local_site_id(%{"local_site_id" => local_site_id}), do: local_site_id
     defp parse_local_site_id(_), do: 1
 
+    defp validate_replication_mode(changeset, "0") do
```

Comment:
We should rename the methods adding a `maybe_` here, as this methods doesn't validate in all the cases